src/sidebar/store/create-store.js

```diff
@@ -203,9 +270,6 @@ export function makeAction(reducers, type, payload) {
   return { type, ...payload };
 }
 
-// The properties of the `config` argument to `createStoreModule` below are
-// declared inline due to https://github.com/microsoft/TypeScript/issues/43403.
```

Comment:
This issue has been fixed, but I didn't think the code benefitted from being changed as a result.